src/cursor.js

```diff
@@ -239,7 +239,9 @@ export default class Cursor {
   // (see: http://www.w3.org/DOM/faq.html#ownerdoc)
   createElement (tagName, attributes = []) {
     const element = this.win.document.createElement(tagName)
-    for (const {attributeName, attributeValue} of attributes) {
+    // I'd like to use for .. of with Object.entries() here but no IE support...
+    for (const attributeName of Object.keys(attributes)) {
```

Comment:
Can't you just use `for key in object`?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

```
for (const key in attributes) element.setAttribute(key, attributes[key])
```